Dockerfile.rhel7

```diff
@@ -3,7 +3,7 @@ WORKDIR /go/src/github.com/openshift/openshift-tuned
 COPY . .
 RUN make build
 
-FROM registry.svc.ci.openshift.org/ocp/4.0:base
+FROM registry.svc.ci.openshift.org/ocp/4.3:base
```

Comment:
nit: This doesn't appear related to the bug fixed and described in the commit message.  It'd be better to split this into another commit.

It's obvious enough though, so you don't need to redo it this time ...

Reply:
(context: i'm reviewing backport PRs as patch manager this week)

Reply:
Understood and thanks, Russell.  It would be great if this could merge this week (customer case).  Let me know if I need to change anything this time round.  https://github.com/openshift/cluster-node-tuning-operator/pull/128 is associated with the case and needs to go to 4.3 too.